apps/emqx/src/emqx_channel.erl

```diff
@@ -1092,6 +1107,23 @@ maybe_update_expiry_interval(
 maybe_update_expiry_interval(_Properties, Channel) ->
     Channel.
 
+process_broker_disconnect(
```

Comment:
nit: I still keep my opinion. In this module is not concerned with what is called broker_disconnect, so, it would be confusing.

Or would it be better to change the name of the function?